files/install-rpm-packages.yaml

```diff
@@ -8,7 +8,7 @@
       - python3-flask
       - python3-ipdb  # for easy debugging
       - python3-jsonschema
-      #- python3-ogr  # temporary install 0.8.0 from PyPI
+      - python3-ogr
```

Comment:
Do you want to release new OGR to have your PR released?

Reply:
not needed, thanks